src/rddlSDKAPI.c

```diff
@@ -101,15 +109,15 @@ void runRDDLSDKMachineAttestation(const char* machineCategory, const char* manuf
 
 void runRDDLSDKNotarizationWorkflow(const char* data_str, size_t data_length){
   Google__Protobuf__Any anyMsg = GOOGLE__PROTOBUF__ANY__INIT;
-  clearStack();
+  abstClearStack();
   if( !getPlntmntKeys() )
     return;
 
   if( !hasMachineBeenAttested() )
     return;
 
   size_t data_size = data_length;
-  uint8_t* local_data = getStack( data_size+2 );
+  uint8_t* local_data = abstGetStack( data_size+2 );
```

Comment:
malloc is used in linux fir getStack. In the following ifdef LINUX_MACHINE, free must be called for local_data too.